### PR TITLE
Update plain-language.md

### PR DESCRIPTION
**Pull request by thejordanwood**

### CODE REVIEW

**Comment by thejordanwood** [comment]
Approvals for content were given in [this Slack thread](https://civicactions.slack.com/archives/CCX8C4MCY/p1723146152694689).

**Review by dmundra — CHANGES_REQUESTED** [review]
@thejordanwood looks like the only change is to use the marker style '*' instead of '-' and then this can be merged. Can you make that change here and in #771?

**Comment by thejordanwood** [comment]
@dmundra I just did that on multiple PRs. Let me go back and correct them!

**Comment by thejordanwood** [comment]
@dmundra Changes have been made

File: _guide/plain-language.md [hunk 1]
```diff
@@ -17,112 +17,73 @@ roles:
...
+* Ask your users to read your content aloud. This can help you identify what words they may struggle with. Users can also rate the clarity of the language. 
+* Ask your users to paraphrase what they read to gauge their understanding. 
 
-People visit your website to do something: answer a question or complete a specific task. Ask yourself: what are the top questions my target reader wants to know? How can I answer it quickly?
+## Resources
 
-### Choose words your reader uses
+* [Federal Plain Language guidelines](https://www.plainlanguage.gov/guidelines/)
+* [Plain Language Association International guidelines](https://plainlanguagenetwork.org/plain-language/what-is-plain-language/)
+* [WCAG 2.1 Reading level success criterion ](https://www.w3.org/WAI/WCAG21/Understanding/reading-level.html)
```
Comment (dmundra) [inline _guide/plain-language.md:80]: @thejordanwood looks like an extra space here before the `]`

Comment (thejordanwood) [reply _guide/plain-language.md]: Fixed! Thanks

**Review by dmundra — APPROVED** [review]

**Pull request merged**